src/nwb_benchmarks/core/_streaming.py

```diff
@@ -183,6 +190,46 @@ def read_hdf5_nwbfile_ros3(s3_url: str, retry: bool = True) -> Tuple[pynwb.NWBFi
     return (nwbfile, io, retries)
 
 
+def create_lindi_reference_file_system(s3_url: str, outfile_path: str):
+    """
+    Create a lindi reference file system JSON cache file for a given HDF5 file on S3 (or locally)
+
+    The output_file path should end in the '.lindi.json' extension
+    """
+    # Create a read-only Zarr store as a wrapper for the h5 file
+    store = lindi.LindiH5ZarrStore.from_file(s3_url)
+    # Generate a reference file system
+    rfs = store.to_reference_file_system()
+    # Save it to a file for later use
+    with open(outfile_path, "w") as f:
+        json.dump(rfs, f, indent=2)
```

Comment:
Use convenience function which standardizes the `json.dump` call. (The lindi `README` needs to be updated)

```suggestion
    # Generate a reference file system and write it to a file
    store.write_reference_file_system(outfile_path)
```